keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/KeelRepository.kt

```diff
@@ -132,27 +132,25 @@ interface KeelRepository {
 
   fun getResourcesByApplication(application: String): List<Resource<*>>
 
-  fun getResourceSummaries(deliveryConfig: DeliveryConfig): List<ResourceSummary>
-
   fun storeResource(resource: Resource<*>)
 
   fun deleteResource(id: String)
 
-  fun applicationEventHistory(application: String, limit: Int): List<ApplicationEvent>
+  fun applicationEventHistory(application: String, limit: Int = DEFAULT_MAX_EVENTS): List<ApplicationEvent>
```

Comment:
why the `DEFAULT_MAX_EVENTS` was added?

Reply:
Because you can’t query events from the database without a limit, and the controller receives it as a parameter, which needs to be passed down all the way to the repository.

Reply:
Sorry, I misread your comment. We may not need the default here, actually. Let me check.